content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -191,6 +191,30 @@ details are abstracted away. That abstraction and separation of concerns simplif
 system semantics, and makes it feasible to extend the cluster's behavior without
 changing existing code.
 
+## Pod update and replacement
+
+As mentioned in the previous section, when the Pod template for a workload
+resource is changed, the controller will create new Pods based on the updated
+template instead of updating or patching the existing Pods.
+
+Kubernetes doesn't prevent you from managing Pods directly. It is possible to
+update a Pod in place. However, the Pod update operation has some limitations:
+
+- Most of the metadata about a Pod is immutable. For example, you cannot
+  change the `namespace`, `name`, `uid`, `creationTimestamp` fields, you can
```

Comment:
nit?: For example, you cannot change the `namespace`, `name`, `uid`, or `creationTimestamp` fields.
nit: The `generation` field is unique. This field only accepts updates that increment the field's current value?